src/main/java/com/fasterxml/jackson/databind/DatabindException.java

```diff
@@ -39,4 +39,11 @@ protected DatabindException(String msg, Throwable rootCause) {
      * current path
      */
     public abstract void prependPath(Object referrer, String fieldName);
+
+    /**
+     * Method called to prepend a reference information in front of
+     * current path
+     */
+    @Deprecated
```

Comment:
For context: this does not make sense due to `DatabindException` being the "new" exception to use (and only one in 3.0) -- `JsonMappingException` will be removed from 3.0.